partiql-logical-planner/src/graph.rs

```diff
@@ -60,7 +61,8 @@ impl<'a> Normalize<'a> {
     /// 'Normalize' a series of node/edge/subpath match elements into a rigid pattern of
     ///    (<node> ( <edge> <node>)*)
     pub fn normalize(mut self, elements: Vec<MatchElement>) -> Result<Vec<PathPattern>, String> {
-        self.do_normalize(elements)
+        let path_mode = PathMode::Walk; // TODO replace with input to function?
+        self.do_normalize(elements, path_mode)
```

Comment:
Should `Walk` be hard-coded here? Perhaps this is causing the code coverage miss for the other `PathMode`s -- https://app.codecov.io/gh/partiql/partiql-lang-rust/pull/575?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=partiql

Reply:
It shouldn't matter, as, if you look inside `do_normalize`, this hard-coded mode only applies to single-node and single-edge matches for which mode doesn't matter.

I've added some additional tests to make sure the appropriate lines are hit.

Reply:
Ah I see now